README.md

```diff
@@ -1,306 +1,46 @@
+[![Zarf Documentation](https://img.shields.io/badge/web-zarf.dev-775ba1)](https://zarf.dev/)
+[![Go version](https://img.shields.io/github/go-mod/go-version/defenseunicorns/zarf?filename=go.mod)](https://go.dev/)
 
 <img align="right" alt="zarf logo" src=".images/zarf-logo.png"  height="256" />
 
 Zarf simplifies the setup & administration of Kubernetes clusters, cyber systems & workloads that support DevSecOps "across the [air gap](https://en.wikipedia.org/wiki/Air_gap_(networking))."
 
-It provides a static go binary (can run anywhere) CLI that can pull, package, and install all the things your clusters need to run and any necessary resources to standup infrastructure such as Terraform. Zarf also caches downloads (for speed), hashes packages (for security), and can _even install the Kubernetes cluster itself_ if you needed.
-
-Zarf runs on [a bunch of operating systems](./docs/supported-oses.md). It aims to support configurations ranging from "I want to run one, simple app" to "I need to support & dependency control a _bunch_ of internet-disconnected clusters."
+It provides a static go binary (can run anywhere) CLI that can pull, package, and install everything a cluster needs to run alongside any necessary infrastructure resources like Terraform. Zarf also caches downloads (for speed), hashes packages (for security), and can _even install the Kubernetes cluster itself_ if you needed.
 
-Zarf was theorized and initially demonstrated in Naval Postgraduate School research:
+Zarf runs on [a bunch of operating systems](./docs/supported-oses.md), and aims to support configurations ranging from "I want to run one, simple app" to "I need to support & dependency control a _bunch_ of internet-disconnected clusters."
 
-[A DEVSECOPS APPROACH FOR DEVELOPING AND DEPLOYING CONTAINERIZED CLOUD-BASED SOFTWARE ON SUBMARINES](https://calhoun.nps.edu/handle/10945/68688).
+Zarf was initially theorized and demonstrated in research from Naval Postgraduate School which you can read [here](https://calhoun.nps.edu/handle/10945/68688).
 
+## Demo
 
 [![asciicast](https://asciinema.org/a/475530.svg)](https://asciinema.org/a/475530)
 
-## Why is Zarf Needed?
-Most of the software ecosystem assumes your systems have access to the internet. The world (for good reasons) has become more and more dependent upon Software as a Service (SaaS), which assumes a robust connection to the internet and a willingness to inherently trust 3rd party providers. Although this makes sense for most of the world, certain secure systems must operate either fully disconnected, semi-disconnected, or might need the ability to disconnect in case of emergencies (like while under an active cyber attack). Although only a small percentage of systems, these secure systems make up some of the most vital systems globally, such as Aerospace and Defense, Finance, Healthcare, Energy, Water, Sewage, and many Federal, Local, and State Government systems.  
-
-## Explain Zarf Like I'm Ten(ish)
```

Comment:
I really like the how much easier it is to consume the README when it's simplified down like this.
Wanted to pose thoughts on two possible additions:

1. Do we think it would be useful to keep an ELI5/10 section? 
    - I think it might be useful for people to get a super quick high-level summary before they make the 'commitment' to click over to the website/docs.
2. Thoughts on adding a `## Getting Started` section?
    - Right now we link to the docs but we con't have a very clear call to action "If you're interested, here's what your next steps could be". Even if it's a short sentence or two that ends up linking to a specific page on the docs website, I think it might be useful information.

Reply:
1. I want to kill explain it like I'm bleh section here, I think it fits better on the main site or docs personally. 

2. Call to action could be link to "try now" / "QuickStart"

Reply:
👍 

1. Maybe we could at migrate a couple sentences from the ELI5 section to the first couple paragraphs of the README. 


Right now, even though we mention the phrase 'air gap', all we're saying to describe what Zarf does boils down to `Its a static go binary that can pull, package, and install all the things your clusters need to run` and I feel like we're missing some of the 🪄 magic that makes Zarf a tool that really helps deploy things across an air gap.

Reply:
Yeah I was going to play with some rewording on that part in my comments. Also bullet point what you get from zarf would be good as @Madeline-UX brought up in another PR.

Reply:
I like the idea of a quick start, and re-titled the "Docs" section "Getting Started" since that is what we want them to do from that section.